### PR TITLE
Users/v andivanov/node 10 appcenter test v1

### PR DESCRIPTION
**Pull request by mpodriezov**
**Task name**: <Name of changed or new pipeline task>

**Description**: Migrated to Node10 for AppCenterTestV1 task. The original PR is here [#15700](https://github.com/microsoft/azure-pipelines-tasks/pull/15700). It was done as fork by @AndreyIvanov42 . I fixed conflicts and fixed CG bugs. 

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AppCenterTestV1/appcentertest.ts [hunk 1]
```diff
@@ -209,7 +209,7 @@ async function setTestRunIdBuildPropertyAsync(testRunId: string) {
         let auth = token.length == 52 ? apim.getPersonalAccessTokenHandler(token) : apim.getBearerHandler(token);
         let vsts: apim.WebApi = new apim.WebApi(url, auth);
         let conn: lim.ConnectionData = await vsts.connect();
-        let buildApi = vsts.getBuildApi();
+        let buildApi = await vsts.getBuildApi();
```
Comment (sergei-fedorov) [inline Tasks/AppCenterTestV1/appcentertest.ts:212]: Why we use `await` here?

File: Tasks/AppCenterTestV1/task.loc.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "Minor": 205,
     "Patch": 2
```
Comment (sergei-fedorov) [inline Tasks/AppCenterTestV1/task.loc.json:16]: Need to bump version

**Review by sergei-fedorov — CHANGES_REQUESTED** [review]
Thanks, looks good, found few remarks.

+ I think we should fix failing ci
and probably test manually

File: Tasks/AppCenterTestV1/appcentertest.ts (hunk 1 above)
Comment (AndreyIvanov42) [reply Tasks/AppCenterTestV1/appcentertest.ts:212]: This method returns a promise

image

Comment (AndreyIvanov42) [reply Tasks/AppCenterTestV1/appcentertest.ts:212]: The link in the online editor leads to the old version of the type definition
image

**Comment by sergei-fedorov** [comment]
Just for the record diff with master is shown incorrectly. Changes in `common-npm-packages` and `Tasks/DownloadGitHubNugetPackageV1` match master. There are only changes in `AppCenterTestV1`.

**Comment by sergei-fedorov** [comment]
DO NOT MERGE this one. Testing showed regression. Target date for the fix July, 19.

**Comment by sergei-fedorov** [comment]
New PR https://github.com/microsoft/azure-pipelines-tasks/pull/16597

**Pull request closed without merging**